site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -2,370 +2,117 @@
-the instructed logic; it is only loaded when it is needed and unloaded when it goes idle. The
-service worker has access to all the [Chrome APIs][section-apis], as long as it declares the
-required permissions in the `manifest.json`.
-
-An extension can only have a single service worker. To import further code, the service worker can be declared as an [ES Module][webdev-imports] by specifying `"type": "module"` in the manifest `"background"`.
-
-See [Manage events with service workers][docs-service-worker] to learn more. 
-
-### Content scripts {: #contentScripts }
-
-Content scripts allow extensions to inject logic into a page in order to read and modify its
-contents. A content script contains JavaScript that executes in the context of a page that has
-been loaded into the browser.
-
-Content scripts can communicate with their parent extension by exchanging [messages][docs-messages]
-and storing values using the [storage][api-storage] API.
-
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/466ftDp0EXB4E1XeaGh0.png", alt="Shows a communication
-path between the content script and the parent extension", height="316", width="388" %}
-
-See [Understanding content scripts][docs-content-scripts] to learn more.
-
-### UI elements {: #pages }
-
-An extension's user interface should be purposeful and minimal. The UI should customize or enhance
-the browsing experience without distracting from it. 
-
-The following is a list of the most common UI examples:
-
-- An [action click][docs-click] event.
-- A [popup][docs-popup].
-- A [context menu][docs-context-menu].
-- An [omnibox][docs-omnibox].
-- A [keyboard shortcut][docs-commands].
-- Desktop [notifications][api-notif].
-- [Text-to-speech][api-tts].
-- A custom UI injected [into a page][docs-content-scripts].
-
-See [Design the UI of a Chrome extension][docs-ui], to learn more.
-
-### Options page {: #optionsPage }
-
-Just as extensions allow users to customize the Chrome browser, the options page enables
-customization of the extension. Options can be used to enable features and allow users to choose
-what functionality is relevant to their needs.
-
-Users can access the options page via [direct link][docs-link-options] or in the context menu of the
-extension toolbar. The following is an example of the Google Dictionary extension. 
-
-{% Columns %}
-
-{% Column %}
-
-<figure>
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/Mz7GV76tFkzxRlb7Pq6e.png", 
-alt="Options page link in the UI", width="800", height="299" %}
-  <figcaption>
-    Link to the Options page.
-  </figcaption>
-</figure>
-
-{% endColumn %}
-
-{% Column %}
-
-<figure>
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/BM11QeGCThsUNTlsZbAe.png", 
-alt="Context Menu Options page", width="357", height="222" %}
+The manifest is the configuration file of a Chrome extension. It is a required JSON file that must be located at the root of the project. It provides the browser with a blueprint of the extension, with important information such as:
```

Comment:
I added a link to [Development basics - Structure](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#structure). I think this might be sufficient. If not, we can circle back later.